common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -69,6 +71,16 @@ class InteractForm extends Component<Props, State> {
     if (nextProps.currentTo.raw !== prevProps.currentTo.raw) {
       nextProps.resetState();
     }
+
+    if (nextProps.currentUnit !== prevProps.currentUnit && nextProps.currentUnit) {
```

Comment:
Doing this in `componentDidMount` breaks the app state. I had to wait until `UNIT_META_SET`